extensions/ql-vscode/src/view/model-editor/ModelAlertsIndicator.tsx

```diff
@@ -0,0 +1,55 @@
+import { styled } from "styled-components";
+import type { ModeledMethod } from "../../model-editor/modeled-method";
+import type { ModelEvaluationRunState } from "../../model-editor/shared/model-evaluation-run-state";
+import type { ModelEditorViewState } from "../../model-editor/shared/view-state";
+
+const ModelAlertsButtonContainer = styled.div`
+  min-height: calc(var(--input-height) * 1px);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const ModelAlertsButton = styled.button`
+  color: var(--vscode-editor-foreground);
+  background-color: var(--vscode-input-background);
+  border: none;
+  border-radius: 40%;
+  cursor: pointer;
+`;
```

Comment:
I copied this styling from the [`UsagesButton`](https://github.com/github/vscode-codeql/blob/1eba41affb9ebb4aaafe48109d72ccc37aac11d1/extensions/ql-vscode/src/view/model-editor/MethodRow.tsx#L52-L58), though I was wondering if we could use the [`VSCodeBadge`](https://github.com/microsoft/vscode-webview-ui-toolkit/blob/main/src/badge/README.md) in both places? 🤔

Reply:
If `VSCodeBadge` looks and works the same I think switching both places to use it makes sense. Probably a second PR if that's ok.

I think I wrote the other example and I probably just didn't know the `VSCodeBadge` component existed or forgot to look for it. I don't remember any reason I didn't use it.

Reply:
Are we planning to switch to a `VSCodeBadge` in a separate PR?

Reply:
Yep, I'll do that in a follow-up PR 📛